docs/docs/configuration-parameters.md

```diff
@@ -61,7 +61,9 @@ It is sufficient to specify just one of the `zk` or `masters` field. If both are
 
 `externalon` is a boolean field that controls whether Mesos-DNS serves requests outside of the Mesos domain. The default value is `true`. 
 
-`SOAMname` is the MNAME field in the SOA record for the Mesos domain. The format is `mailbox.domain`, using a `.` instead of `@`. For example, if the email address is `root@ns1.mesos`, the `email` field should be `root.mesos-dns.mesos`. For details, see the [RFC-1035](http://tools.ietf.org/html/rfc1035#page-18). The default value is `root.ns1.mesos`. 
+`SOAMname` specifies the domain name of the name server that was the original or primary source of data for the configured domain. The default value is `ns1.mesos`.
```

Comment:
Our code seems wonky.

Our default values (and docs) specify relative names:
https://github.com/mesosphere/mesos-dns/blob/master/records/config.go#L76

Then we later **assume** that the config should use fully qualified names
(even if a user doesn't want it!):
https://github.com/mesosphere/mesos-dns/blob/master/records/config.go#L125

So basically there's no way to use a relative name. Is that OK?


On Mon, Oct 5, 2015 at 4:16 PM, Tomás Senart <notifications@github.com>
wrote:

> In docs/docs/configuration-parameters.md
> <https://github.com/mesosphere/mesos-dns/pull/296#discussion_r41191635>:
>
> > @@ -61,7 +61,9 @@ It is sufficient to specify just one of the `zk` or `masters` field. If both are
> >
> >  `externalon` is a boolean field that controls whether Mesos-DNS serves requests outside of the Mesos domain. The default value is `true`.
> >
> > -`SOAMname` is the MNAME field in the SOA record for the Mesos domain. The format is `mailbox.domain`, using a `.` instead of `@`. For example, if the email address is `root@ns1.mesos`, the `email` field should be `root.mesos-dns.mesos`. For details, see the [RFC-1035](http://tools.ietf.org/html/rfc1035#page-18). The default value is `root.ns1.mesos`.
> > +`SOAMname` specifies the domain name of the name server that was the original or primary source of data for the configured domain. The default value is `ns1.mesos`.
>
> Internally all names are fully qualified, but we haven't been requiring
> users to do it explicitly.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mesosphere/mesos-dns/pull/296/files#r41191635>.
>